README.md

```diff
@@ -1,167 +1,141 @@
+## Installation
 
-## Install
+### 1. Setup Python Environment
 
-From within the python environment described in the [Environment](#environment) section, run the command:
+We recommend using a virtual environment:
 
-`pip install git+https://github.com/309thEDDGE/opal-release-downloader.git`
+```sh
+python -m venv opal_env
+source opal_env/bin/activate  # On Windows, use `opal_env\Scripts\activate`
+```
 
-_Note: On windows be sure to use_ `python -m pip install ...`.
+### 2. Install OPAL Release Downloader
 
-This command will pull the source from github, then build and install the python package, including dependencies. It will place several scripts on your path in the active environment. Scripts are described in [Download and verify artifacts](#download-and-validate-artifacts).
+```sh
+pip install git+https://github.com/309thEDDGE/opal-release-downloader.git
+```
 
-## Usage
+## Download and Verify OPAL Artifacts
 
-### Runtime arguments: 
+1. **Navigate to an empty directory** where OPAL artifacts will be stored.
+2. **Ensure you are in the correct Python environment.**
+3. **View script options:**
+   ```sh
+   download_opal_artifacts -h
+   ```
+4. **Download artifacts:**
+   ```sh
+   download_opal_artifacts opal-deployment-artifacts
+   ```
+   This downloads and verifies required files into `opal_artifacts/`.
 
-* must be obtained from OPAL developers prior to use
-* S3 bucket name in which OPAL artifacts reside, referred to as `<bucket name>`
-* Release tag of available OPAL artifacts version, in the form of a date _YYYY.MM.DD_, referred to as `<release_tag>`
+## OPAL Deployment
 
-### Download and validate artifacts:
+### 1. Copy Artifacts
 
-* Navigate to an empty directory in which OPAL artifacts shall reside
-* (if applicable) Ensure that the active environment is the one in which opal-release-downloader was installed
-* View the help menu of the download script: `download_opal_artifacts -h`
-* Execute the download script and pass arguments as described in the help menu: `download_opal_artifacts <bucket_name> ...`
-* `<bucket_name>` can be omitted to automatically grab the latest tag
-* See `list_opal_artifacts -h` to view available tags
-* Do not use the flags `--no-docker` or `--no-rhel` unless you are an expert
-* Some of the compressed images are several GBs in size. The download and verification process can take over an hour depending on internet connection and computer performance.
-* If the command runs without error, the `opal_artifacts` directory contains all of the artifacts required to deploy OPAL
+Move `opal_artifacts/` to your target deployment machine.
 
-## OPAL Deployment
+### 2. Install OPAL
 
-### Install Red Hat Linux
```

Comment:
Leave this in, but mention that the redhat install is only applicable if you don't have linux already installed, also the --no-rhel flag needs to be omitted from the download_opal_artifacts if rhel is to be included during the download step.